adoc/admin-security-ldap.adoc

```diff
@@ -2,7 +2,122 @@
 
 == Configuring An External LDAP Server
 
-{productname} supports user authentication via external LDAP server like 389 Directory Server and Active Directory by updating the builtin Dex LDAP connector configuration.
+{productname> supports user authentication via external LDAP server like 389 Directory Server and Active Directory by updating the builtin Dex LDAP connector configuration.
+
+=== Deploy an External 389 Directory Server with Automatically Generated Self-signed Certificate
+
+The 389 Directory Server image `registry.suse.com/caasp/v4/389-ds:1.4.0` will automatically generate self-signed certificate and key. 389 Directory Server can be deployed with customized configuration and container command:
+
+. Prepare customized 389 Directory configuration to enter in Terminal:
++
+----
+INSTANCE_NAME=localhost                         # Directory Server Name
+ROOT_DN="cn=admin,dc=example,dc=org"            # Admin Account
+ROOT_PASSWORD=                                  # Admin Password
+SUFFIX="dc=example,dc=org"                      # Suffix
+DATA_DIR=<PWD>/389_ds_data                      # Directory Server Data
+IMAGE=registry.suse.com/caasp/v4/389-ds:1.4.0   # Container Image 
+----
+
+. Execute `docker` command to deploy 389 Directory Server. This will start non-TLS port (389) and TLS port (636) together with automatically self-signed certificate and key.
```

Comment:
```suggestion
. Execute `docker` command to deploy 389 Directory Server. This will start a non-TLS port (389) and a TLS port (636) together with an automatically self-signed certificate and key.
```